### PR TITLE
Topic Map Marker, closed #70

### LINKED ISSUE #70: Topic Map Marker
Topic Marker are not getting displayed on the map in the specific district.
I created a specific database dump with 3 topics and locations that contain geodata in lindenthal.
They also contain post codes ( 50933 & 50858 ) for their specific region

[database_2019-10-21.sql.zip](https://github.com/nextsolx/ratsinformationssystem/files/3750942/database_2019-10-21.sql.zip)

### PR DESCRIPTION
**Pull request by krausweb**
- fixed display marker, only if exist related topic
- fixed display the polygon, marker, popup according to the design
- fixed lazy load block layout
- fixed the layout with long meeting list name on the member detail page
- fixed the error bug with the empty location on the theme detail page
- updated readme
- refactored styles, naming, minor changes

---

**Test cases**
City -> Lindenthal -> Lindenthal
or
City -> Lindenthal -> Junkersdorf

### CODE REVIEW

File: resources/js/components/Map/MapAside.vue [hunk 1]
```diff
@@ -16,7 +16,7 @@ export default {
     data () {
         return {
             subTitle: 'Aktualle Themen',
```
Comment (dclermont90) [inline resources/js/components/Map/MapAside.vue]: Typo at Aktualle -> Aktuelle

**Comment by dclermont90** [comment]
I cant see the topic markers on the map, this is how it looks for me.
Bildschirmfoto 2019-10-24 um 10 48 01

**Comment by krausweb** [comment]
@dclermont90, please re-check
Implemented themes markers (It works with test DB data, see Topic Map Marker #70)

**Comment by dclermont90** [comment]
@MBeckers Looks good to me.

**Pull request merged**